Scripts/script-ExtractDomainFromUrlFormat.yml

```diff
@@ -0,0 +1,32 @@
+commonfields:
+  id: ExtractDomainFromUrlAndEmail
+  version: -1
+name: ExtractDomainFromUrlAndEmail
+script: |-
+  from tld import get_tld
+  from validate_email import validate_email
+  import re
+
+
+  inp = demisto.args()['input']
+
+  is_email = validate_email(inp)
```

Comment:
You are running this script only on urls

Reply:
both urls and emails